packages/foreman/foreman-selinux/foreman-selinux.spec

```diff
@@ -68,12 +64,7 @@ SELinux policy module for Foreman
 
 %build
 # determine distribution name and version
-%if 0%{?rhel} >= 6
 %define distver rhel%{rhel}
-%endif
-%if 0%{?fedora} >= 18
-%define distver fedora%{fedora}
```

Comment:
Unrelated, but this reminded me https://github.com/theforeman/foreman-selinux/pull/148 is still open.